### PR TITLE
Add Contributing Information

### PR DESCRIPTION
**Pull request by GavinFigueroa**
Creates documentation for how to contribute to the project.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Maintainers
+- Gavin Figueroa <gavin_figueroa@homedepot.com>
```
Comment (billimek) [inline MAINTAINERS.md]: Can you please include your github `@username` here as well?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Contributions Welcome
+
+First off, thank you for considering contributing to the Github-Webhook-Resource! This resource is a simple project, but has plenty of room for improvement. Many features of Github's webhook implementation are yet to be built into the Github-Webhook-Resource, and all contributions are welcome.
+
+If you're just looking for quick feedback for an idea or proposal, feel free to open an
+[issue](https://github.com/homedepot/github-webhook-resource/issues).
```
Comment (billimek) [inline CONTRIBUTING.md]: How about linking this to the 'create new issue' page instead?  https://github.com/homedepot/github-webhook-resource/issues/new

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+2. Create your contribution, meeting all
+   [contribution quality standards](#contribution-quality-standards)
+3. [Create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/)
+   against the master branch of the Github-Webhook-Resource repository.
+4. Add a reviewer to your pull request. Work with your reviewer to address any comments and obtain an approval.
+   To update your pull request amend existing commits whenever applicable and
+   then push the new changes to your pull request branch.
+5. Once the pull request is approved, one of the [maintainers](MAINTAINERS.md). will merge it.
+
+## Request for Comments
```
Comment (billimek) [inline CONTRIBUTING.md]: For this particular repo, I suggest we remove the entire **Request for Comments** section

Comment (GavinFigueroa) [reply CONTRIBUTING.md]: I think I understand why this is a good idea, but could you share your thoughts behind this change?

**Pull request merged**